src/components/ResultsItem/ResultsItem.tsx

```diff
@@ -19,6 +19,7 @@ export type ResultItemProps = {
   meta?: ResultItemMetaProps;
   text?: string;
   title: string;
+  nodeType?: string;
   type?: 'content' | 'file' | 'taxonomy_term';
```

Comment:
wondering if the naming of these two parameters is confusing now... what would `type` (content/file/taxonomy_term) be called in Drupal? File type? Content type?

Reply:
I'm trying to be really specific and trying to align with what you get from solr as well, that's why I called it nodeType.

Reply:
yeah with you there, wondering more about the name of the `type` parameter now that we have to "type" params...

Reply:
would it be better as `contentType`? or `fileType`?

Reply:
actually let's park this and address is another PR (if needs be)